.travis.yml

```diff
@@ -24,7 +24,7 @@ install:
 after_success: 
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       yarn coverage;
-      if [[ $TRAVIS_BRANCH == "development" ]] || [[ $TRAVIS_BRANCH == "master" ]]; then
+      if [[ $TRAVIS_BRANCH == "development" ]] || [[ $TRAVIS_BRANCH == "master" ]] && [[ "$TRAVIS_PULL_REQUEST" = "false"]] ; then
```

Comment:
No need for quotes around the var name.

There's a stray space after the condition/before the semicolon.

Reply:
Follow [their example](https://docs.travis-ci.com/user/pull-requests/#Pull-Requests-and-Security-Restrictions), I'm going to keep the quotes. I'll get rid of the space.